Sources/JWTKeychain/Provider/UserResponse.swift

```diff
@@ -1,3 +1,4 @@
+import Sugar
```

Comment:
Can this be removed?

Reply:
It is required for `PublicRepresentable`